02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+[
+    {
+        "id": 1,
+        "question": "The client has her 19 year old grandchild living with her. She says she does her own thing and they don't really share food. Does the grandchild need to be included?",
+        "guru_cards": [
+            "Who are mandatory HH members for food stamps?"
+        ],
+        "recall": 1.0,
+        "extra_cards": 3,
+        "derived_questions": [
+            "Does the grandchild live with you?",
+            "Is the grandchild financially dependent on you?",
+            "Do you provide regular support for the grandchild?"
+        ],
+        "all_retrieved_cards": {
+            "Can children under 18 years of age ever be certified as a separate household from other adults living in the home?": {
+                "score_sum": 0.7314764598621692,
+                "derived_questions": [
```

Comment:
A subset of the total derived questions (line 10), including the derived questions that resulted in retrieval of this specific card.